application.js

```diff
@@ -1,7 +1,8 @@
 var mbaasApi = require('fh-mbaas-api');
 var express = require('express');
 var mbaasExpress = mbaasApi.mbaasExpress();
-var cors = require('cors');
+require('./lib/db.js')();
+//var cors = require('cors');
```

Comment:
Right now, the requests are being .pipe'd as part of the proxy, so CORS breaks this. Honestly I'm not sure we're getting great benefit from .pipe-ing right now though, so I'd like to bring it back in